meta-lmp-base/recipes-support/fioconfig/fioconfig_git.bb

```diff
@@ -11,7 +11,7 @@ SRC_URI = "git://${GO_IMPORT};protocol=${GO_IMPORT_PROTO};branch=main \
 	file://fioconfig.path \
 	file://fioconfig-extract.service \
 "
-SRCREV = "0b14975bbcfa85590432a9e71b1eb0125d81a92a"
+SRCREV = "c9469923f88cec496a9abce194d698fda0f3573d"
```

Comment:
Not sure if we usually do a _commit hash upgrade_ in a separate commit, or it is fine to combine it with code changes.

@ricardosalveti can give the guidance on this.

Reply:
As in this case the only change involved is what allowed the recipe / packaging change, it is fine to have in a single commit.

Reply:
In this case, the SRCREV bump is aligned with the changes. since the changes are minor on `fioconfig` side and here, I don´t think it requires an extra PR.